bftengine/src/bftengine/ReplicasInfo.cpp

```diff
@@ -55,6 +55,7 @@ ReplicasInfo::ReplicasInfo(const ReplicaConfig& config,
                            bool dynamicCollectorForPartialProofs,
                            bool dynamicCollectorForExecutionProofs)
     : _myId{config.replicaId},
+      _isRoReplica{(_myId >= config.numReplicas) && (_myId < config.numReplicas + config.numRoReplicas)},
```

Comment:
There is a configuration variable in config that specifies if the replica is a read-only, please use it instead

Reply:
Done